### PR TITLE
How to infer interaction strength

### PR DESCRIPTION
**Pull request by BenMerSci**
Initial changes to create first draft pull request

### CODE REVIEW

**Comment by DominiqueCaron** [comment]
You might already have read it, but maybe you can look into this article.: 

[fluxweb: An R package to easily estimate energy fluxes in food webs (Gauzens et al., 2018)](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13109).

It also cites other articles related to inferring interaction strengths

**Comment by tpoisot** [comment]
There is also Berlow et al. (2008) https://www.pnas.org/content/106/1/187 - it's a classic

**Comment by BenMerSci** [comment]
Do we want to lean towards a single type of interaction strength, for example focus on energy fluxes, or do we keep it general by talking about the inference of all kinds of interaction strengths (energy fluxes, frequency of interaction etc. etc.)? @tpoisot @TanyaS08 @gottacatchenall

**Comment by tpoisot** [comment]
> Do we want to lean towards a single type of interaction strength, for example focus on energy fluxes, or do we keep it general by talking about the inference of all kinds of interaction strengths (energy fluxes, frequency of interaction etc. etc.)? @tpoisot @TanyaS08 @gottacatchenall

Good question! I think you can be very explicit about the fact that interaction "strength" can mean a lot of different things!

**Comment by BenMerSci** [comment]
Yes I was going to develop that more on the PR SECTION3/What-is-interaction-strength, but for the inference in itself, I was wondering if we focus on one type, because I think that the methods of infering interation strength can vary greatly depending on which type no?

**Comment by tpoisot** [comment]
Oh I see -- can I suggest that you finish the "what is interaction strength" section first, then we can see how to make this one flow from it?

**Comment by BenMerSci** [comment]
Absolutly!

**Comment by tpoisot** [comment]
I just co-assigned myself to this so I can assist with the writing when #41 is almost done and ready to move to review.

**Comment by BenMerSci** [comment]
As one can see, I'm kind of struggling with this section. I'll try to come up with something this weekend.

File: manuscript.md [hunk 1]
```diff
@@ -42,7 +42,15 @@
 
 ## How are interaction strengths actually inferred?
 
-There are many ways to infer interaction strength, whether it is empirically or theoretically. 
+**General discussion of interaction strength inference method**
+- Interaction strength can either be infered empirically or theoretically [@Berlow2004IntStr]. Theoretical inference can be more mechanistic or with the Lotka-Volterra framework.
```
Comment (gottacatchenall) [inline manuscript.md:46]: I'm thinking relating the beginning of this section to functional response and the early metabolic papers ([Yodzis and Innes 1992](https://www.journals.uchicago.edu/doi/abs/10.1086/285380), [Brown et al 2004](www.dx.doi.org/10.1890/03-9000), [Rooney and McCann's book](https://link.springer.com/chapter/10.1007/978-1-4020-5337-5_2)) could help

**Comment by gottacatchenall** [comment]
merged with #84

**Pull request closed without merging**